AUTHORS.rst

```diff
@@ -24,6 +24,8 @@ Contributors
 
 * Elizabeth Kent <eck@noc.ac.uk> `@lizkent <https://github.com/lizkent>`_
 
+* Jan Marius Willruth <jan.willruth@dwd.de> `@JanWillruth <https://github.com/JanWillruth>`_
+
```

Comment:
You can set your name under `Co-Developer` since you actually changed something in the code.